docs/src/components/ElevationLevel.jsx

```diff
@@ -0,0 +1,52 @@
+import React from "react";
+import styled from "styled-components";
+import theme from "../theme";
+
+export const LevelDiv = styled.div`
+  height: 40px;
+  width: 40px;
+  position: relative;
+  top: 25%;
```

Comment:
Without the container this pushes the box too much down (`left` is still good 👍).
```suggestion
```